app.py

```diff
@@ -226,23 +299,27 @@ def build_record_list(self):
                 if record:
                     self.records.append(record)
 
+            if not record:
+                self.failed_records.append(row)
+
+            self.rows_processed += 1
+
         return self
 
     def get_records(self):
         """
             Deprecated, use `build_record_list` instead.
         """
-        print('get_records is deprecated and will be removed in the future.')
-        print('Please use `build_record_list` instead.')
+        logging.warning('get_records is deprecated and will be removed in the future.')
+        logging.warning('Please use `build_record_list` instead.')
         return self.build_record_list()
 
     def show_records(self):
```

Comment:
deprecate in favor of new method `export_records`, it will dump the records to `Records-FILENAME_TIME.json` instead of the log file